elliptic-curve/src/group_digest.rs

```diff
@@ -0,0 +1,86 @@
+use crate::hash2field::{hash_to_field, ExpandMsg, FromOkm};
+use crate::isogeny::Isogeny;
+use crate::osswu::OsswuMap;
+use ff::Field;
+use group::Group;
+
+/// Convert a set of field elements into a point
+pub trait ConvertToPoint {
+    /// The field element that represents a point
+    type FieldElement: Field;
+    /// The result of converting two field elements to a Point
+    type Output;
+
+    /// Converts two coordinates into a point
+    fn convert_to_point(x: Self::FieldElement, y: Self::FieldElement) -> Self::Output;
+}
+
+/// Method for points to be cleared by the curve cofactor
+pub trait ClearCofactor {
+    /// Multiply this point by the curve cofactor to ensure
+    /// correct subgroup membership
+    fn clear_cofactor(&mut self);
+}
```

Comment:
I think this can be removed in favor of [`CofactorGroup`](https://docs.rs/group/0.11.0/group/cofactor/trait.CofactorGroup.html#tymethod.clear_cofactor)?

Reply:
Yep. Wasn't aware of that method. Thanks for pointing it out.